### DISCUSSION TITLE
Save and persist route state while switching routes

### DISCUSSION THREAD

**Discussion by codegenixdev** [remix-run/react-router#10817] [Q&A]
My project uses react-router-dom@6.15.0. 
In my application, I want when user goes to another route, the previous tab stays open on the top of the screen and when the user gets back again to that page (tab), the state and everything else be remained the same. I do not want for the component to get re rendered every time user goes to another route and I want the state of it to be saved.
Is this possible with react router? Or should I use redux and other state management tools for it?

**Comment by afkcodes** [comment]
did you find anything around this @amirnz026  ?
